onionperf/onionperf

```diff
@@ -145,6 +145,12 @@ def main():
         action="store", dest="tgenpath",
         default=util.which("tgen"))
 
+    measure_parser.add_argument('--client-bridge-lines',
```

Comment:
If this is a file, it is not bridge line specific, so should it not be "client-additional-config"? This can be explained in more detail in the documentation including the requirement for UseEntryGuards.